Airports/EGCC/Sector.txt

```diff
@@ -65,12 +65,12 @@ DISPLAY:CCNTH05:Scottish PC East:CCNTH05
 DISPLAY:CCFIN05:CCFIN05:CCNTH05
 DISPLAY:CCNTH05:CCFIN05:CCNTH05
 COORD:N053.09.54.878:W002.39.12.131
-COORD:N053.20.10.000:W002.50.59.000
-COORD:N053.29.20.000:W002.50.59.000
+COORD:N053.20.00.952:W002.50.58.961
+COORD:N053.28.57.572:W002.51.38.571
 COORD:N053.32.17.000:W002.42.42.000
 COORD:N053.40.08.000:W002.20.57.000
-COORD:N053.36.30.000:W002.03.00.000
-COORD:N053.27.45.000:W002.24.30.000
+COORD:N053.34.40.633:W002.07.38.097
+COORD:N053.27.51.963:W002.24.40.875
 COORD:N053.21.25.000:W002.15.44.000
```

Comment:
```suggestion
COORD:N053.21.25.290:W002.15.44.230
```

Reply:
Make sure to also change this in lines:

- [ ] 34,
- [ ] 39,
- [ ] 47 and
- [ ] 59